example/rollup/lite.pl

```diff
@@ -1,20 +1,37 @@
 #!/usr/bin/env perl
 use Mojolicious::Lite;
-use FindBin;
-BEGIN { unshift @INC, "$FindBin::Bin/../../lib" }
+use Mojo::File 'curfile';
+require lib;
 
-plugin Webpack =>
-  {process => ['svelte'], dependencies => {core => 'rollup', svelte => [qw(rollup-plugin-svelte svelte)]}};
+$ENV{MOJO_WEBPACK_CONFIG}  = "@{[curfile->sibling('rollup.config.js')]}";
+$ENV{MOJO_WEBPACK_DEBUG}   = 1;
+$ENV{MOJO_WEBPACK_VERBOSE} = 0;
+$ENV{MOJO_WEBPACK_LAZY}    = 0;
+
+my $path = "@{[curfile->dirname->dirname->sibling('lib')]}";
+lib->import($path);
+
+plugin Webpack => {
+  process      => ['svelte'],
+  dependencies => {
+    core   => 'rollup',
+    svelte => [qw(rollup-plugin-svelte svelte)]
+  }
+};
 get '/' => 'index';
-app->start;
+
+{
+  local $ENV{PERL5OPT} = "-I$path";
```

Comment:
Can you add a comment for when this environment variable is needed?

Reply:
I've added POD to detail how I ran these examples as a new user - generally the `carton` route. I don't generally install, and the code assumes `M::P::Webpack` isn't installed hence the need for the `use lib ...;`. `@INC` changes aren't propagated unless I'm missing something. Without it `mojo` isn't able to find `Mojolicious::Command::Author::webpack`.

https://github.com/jhthorsen/mojolicious-plugin-webpack/blob/5a397cd07b7fe21d262af4fd1d3d1ca9a6ed6926/lib/Mojolicious/Command/Author/webpack.pm#L40-L45